apps/explorer/src/components/OwnedObjects/index.tsx

```diff
@@ -42,10 +44,10 @@ function getShowPagination(itemsLength: number, currentPage: number, isFetching:
 
 export function OwnedObjects({ id }: { id: string }) {
 	const [filter, setFilter] = useState<string | undefined>(undefined);
-	const [limit, setLimit] = useState(PAGE_SIZES[4]);
+	const [limit, setLimit] = useState(50);
```

Comment:
was this intentional? also it looks like the page limit dropdown is broken (i.e. selecting a limit has no effect)

Reply:
It does not work correctly with `Kiosk` items. I will need to check with Nikhil tomorrow. The drop down works for NFTs items.

Reply:
Remove pagination for Kiosk